rcp/org.testeditor.web.testexecutionlogview/login_test/css/style.css

```diff
@@ -0,0 +1,144 @@
+@import url(http://fonts.googleapis.com/css?family=Vibur);
```

Comment:
That we do not access websites through javascript where code will be executed without our permission.